### ISSUE TITLE
Search required 3 chars to return results, but appear to load with 1 and 2

### ISSUE THREAD

**Issue by KronosTheLate** [jameslittle230/stork#323]
Original issue can be found at https://github.com/MakieOrg/Makie.jl/issues/2004.
Below is a copy of the original issue for convenience.

One and two character searches stay like this:
<img width="252" alt="image" src="https://user-images.githubusercontent.com/61620837/171182376-1c22b21b-9b56-4d03-b2b7-ef5d3baaa16d.png">

The moment i enter a third char, the result pops up.
It should not appear to load when it fact it is waiting for more input.

**Comment by jameslittle230** [comment]
This is an intentional design decision - I felt like when the UI remained unchanged, it didn't signal to users that they should keep typing to get results.

I'm curious what you expected/hoped to see? Did you expect that the UI remain blank until the user types 3 characters? Or did the word "Filtering" mislead you about what Stork is doing?

**Comment by KronosTheLate** [comment]
Fair enough - it is honestly not terrible, and took me all of two seconds to figure out.
I just feel like it would be better to - as you mentioned - show nothing, which 1) does not lie (how I view it, no to be inflamitory), and it does essentially the same. It is just the active word "filtering", along with the three dots that indicate something unfinished, that make me believe that the system is actually filtering something. So yes, nothing would be better.

Optimally, I feel as if the best solution would be to show nothing before 3 chars. However, if the user stops at 1 or 2 chars for say 1 second, prompt the user by showing "Please input 3 character minimum to search".

**Comment by jameslittle230** [comment]
In the version 2.0.0 branch, I'll allow the text displayed to be customizable. I still believe that "Filtering" is the best choice, but I understand that it might not be the best choice for everyone.

**Comment by KronosTheLate** [comment]
Thanks for making it custumizable ^_^

"Filtering" sound to me like the system is activly doing something, and will produce a result when it is done. Do you not agree that "Filtering" just plain wrong, when in fact the system is waiting for input?
